plugins/flatpak/gs-flatpak.c

```diff
@@ -2500,6 +2564,19 @@ gs_flatpak_get_list_for_remove (GsFlatpak *self, GsApp *app,
 		return FALSE;
 	}
 
+	/* If this is the SDK runtime, remove the associated EknServices.
+	 * Ignores EknServices associated with com.endlessm.Platform//eos3.1. */
+	if (g_strcmp0 (ref_name, "com.endlessm.apps.Platform") == 0) {
```

Comment:
As you pointed out recently, since we never remove a runtime from g-s. Even if we have to do so in the future, it's probably a good idea not to carry this code for now (because e.g. if it gets outdated, then when we do implement the removal of runtimes we could forget to update or verify this as long as it compiles...).